### PR TITLE
GRPC enabled arm64v8 dockerfile

### PR DESCRIPTION
**Pull request by ShahanaFarooqui**
GRPC enabled arm64v8 dockerfile
Deleted old arm64v8 dockerfile

### CODE REVIEW

File: contrib/docker/Dockerfile.arm64v8 [hunk 1]
```diff
@@ -63,65 +63,63 @@ RUN apt-get update -qq && \
...
-ENV target_host=aarch64-linux-gnu
-
-ENV AR=${target_host}-ar \
-AS=${target_host}-as \
-CC=${target_host}-gcc \
-CXX=${target_host}-g++ \
-LD=${target_host}-ld \
-STRIP=${target_host}-strip \
-QEMU_LD_PREFIX=/usr/${target_host} \
-HOST=${target_host}
```
Comment (cdecker) [inline contrib/docker/Dockerfile.arm64v8:88]: I'm pretty sure that without these definitions we are actually building the native platform that the host is running. Or are we expecting this to be used from the target platform directly, i.e., not cross-compiling?

Comment (ShahanaFarooqui) [reply contrib/docker/Dockerfile.arm64v8:88]: This Dockerfile will be used within build-release.sh script where we will use Docker's Buildx plugin (using QEMU emulation) to allow building multi-platform/cross-compiled images. 

```
docker buildx build --load --platform linux/arm64 -t "elementsproject/lightningd:$VERSION-$d" -f Dockerfile.$d "${TMPDIR}"
```

I tested this image on arm64v8 Umbrel setup after creating it on amd64 laptop. But I am changing this PR to WIP till build-release PR https://github.com/ElementsProject/lightning/pull/6333 is not finished.

Comment (cdecker) [reply contrib/docker/Dockerfile.arm64v8:88]: Oh, using buildkit `buildx` is clever, I wasn't aware of it. That makes cross-compiling much nicer :-)

**Comment by ShahanaFarooqui** [comment]
Closing this PR in favour of https://github.com/ElementsProject/lightning/pull/6367. Sorry :).

**Pull request closed without merging**